edgepi-thermocouple.html

```diff
@@ -1,132 +1,159 @@
-      updateEditor()
+      updateEditor();
       configTransportSelect.addEventListener("change", updateEditor);
-  }
-	});
+    },
+  });
 </script>
 
 <script type="text/html" data-template-name="thermocouple">
+  <style>
+    * {
+      box-sizing: border-box !important;
+    }
 
-	<style>
-		*{
-			box-sizing: border-box !important;
-		}
+    .form-row {
+      display: flex;
+      align-items: center;
+    }
 
-		.form-row{
-			display: flex;
-			align-items: center;
-		}
+    .form-row > label {
+      margin-top: auto;
+      margin-bottom: auto;
+    }
 
-		.form-row>label{
-			margin-top: auto; margin-bottom: auto;
-		}
+    .form-row.tcp {
+      flex-direction: row;
+      align-items: center;
+      margin-top: -5px;
+      margin-bottom: 10px;
+    }
 
-		.form-row.tcp {
-			flex-direction: row;
-			align-items: center;
-			margin-top: -5px; margin-bottom: 10px;
-		}
+    .tcp-address-input {
+      width: 160px !important;
+      margin-left: 100px !important;
+    }
+    .tcp-port-input {
+      width: 60px !important;
+    }
 
-		.tcp-address-input{
-			width: 160px !important;
-			margin-left: 100px !important;
-		}
-		.tcp-port-input{
-			width: 60px !important;
-		}
+    .tcp-port-label {
+      width: 5px !important;
+      margin: auto 5px;
+    }
+  </style>
 
-		.tcp-port-label{
-			width: 5px !important;
-			margin: auto 5px;
-		}
-	</style>
-	
-	<div class="form-row">
-		<label for="node-input-name"><i class="fa fa-tag"></i> Name:</label>
-		<input type="text" id="node-input-name" placeholder="Name">
-	</div>
-	<div class="form-row transport">
+  <div class="form-row">
+    <label for="node-input-name"><i class="fa fa-tag"></i> Name:</label>
+    <input type="text" id="node-input-name" placeholder="Name" />
+  </div>
+  <div class="form-row transport">
     <label for="node-input-transport">RPC Server:</label>
     <select id="node-input-transport">
       <option value="Local">Local</option>
       <option value="Network">Network</option>
     </select>
   </div>
-  <div class="form-row tcp" >
-    <input class ="tcp-address-input" type="text" id="node-input-tcpAddress" placeholder="IP Address/ Hostname">
+  <div class="form-row tcp">
+    <input
+      class="tcp-address-input"
+      type="text"
+      id="node-input-tcpAddress"
+      placeholder="IP Address/ Hostname"
+    />
     <label class="tcp-port-label" for="node-input-tcpPort">:</label>
-    <input class="tcp-port-input" type="text" id="node-input-tcpPort" placeholder="Port">
+    <input
+      class="tcp-port-input"
+      type="text"
+      id="node-input-tcpPort"
+      placeholder="Port"
+    />
+  </div>
+  <div class="form-row">
+    <label for="node-input-output">Output (&deg;C):</label>
+    <select id="node-input-output">
+      <option value="lin">Linearized Temperature</option>
+      <option value="cj">Cold-Junction Temperature</option>
+      <option value="cj-lin">Cold-Junction & Linearized Temperature</option>
+    </select>
```

Comment:
let's remove these selections. the user does not need to select this.

Reply:
Done